samples/cpp_synchronization/src/main.cpp

```diff
@@ -133,7 +133,7 @@ void coop_thread_entry(void)
 	}
 }
 
-void main(void)
+int main(void)
```

Comment:
Add the return statement.

Reply:
It's not required here, but I've added it as you wish.

Reply:
Thank you.  I've found Zephyr PRs get merged faster if you don't leave anything that people can ask "why is it this way" and decline to approve.  The subtlety that falling off `int main()` is equivalent to `return 0;` (for both C++ and hosted C) is something most people won't know.